alacritty/src/display/content.rs

```diff
@@ -407,31 +410,31 @@ impl<'a> Hint<'a> {
     /// The tuple's [`bool`] will be `true` when the character is the first for this hint.
     fn advance(&mut self, viewport_start: Point, point: Point) -> Option<(char, bool)> {
         // Check if we're within a match at all.
-        if !self.regex.advance(point) {
+        if !self.matches.advance(point) {
             return None;
         }
 
         // Match starting position on this line; linebreaks interrupt the hint labels.
         let start = self
-            .regex
             .matches
-            .get(self.regex.index)
-            .map(|regex_match| max(*regex_match.start(), viewport_start))
+            .matches
+            .get(self.matches.index)
+            .map(|matched| max(*matched.bounds().start(), viewport_start))
```

Comment:
```suggestion
            .map(|match| max(*match.bounds().start(), viewport_start))
```

"Matched" as a noun here seems like a strange choice.